app/admin/collection_points_map.rb

```diff
@@ -0,0 +1,10 @@
+ActiveAdmin.register_page 'CollectionPoints' do
```

Comment:
Did you check that the map is working as expected? I believe that you also need to change views file name